content/vi/docs/reference/glossary/docker.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Docker
+id: docker
+date: 2019-29-11
+full_link: https://docs.docker.com/engine/
+short_description: >
+  Docker là một công nghệ phần mềm thực hiện việc ảo hóa tầng hệ điều hành được gọi là container.
+
+aka:
+tags:
+  - fundamental
+---
+
+Docker là một công nghệ phần mềm thực hiện việc ảo hóa tầng hệ điều hành được gọi là container.
```

Comment:
Must be:
Docker (cụ thể là Docker Engine) là một công nghệ phần mềm thực hiện việc ảo hóa tầng hệ điều hành, còn được gọi là {{< glossary_tooltip text="containers" term_id="container" >}}.